Include/Rover/TrialView.hpp

```diff
@@ -22,15 +22,9 @@ namespace Rover {
 
       //! Random access constant iterator.
       class ConstIterator {
-        public:
-
-          //! Creates a constant iterator.
-          /*
-            \param get_ptr Reference to TrialView::m_get_ptr.
-            \param offset Offset from t.begin().
-          */
-          ConstIterator(const GetPtr& get_ptr, std::size_t offset);
+        friend class TrialView<S>;
```

Comment:
Is this the correct formatting for friend classes?

Reply:
Put the friend specifier in the private section.